stages/test/test_selinux.py

```diff
@@ -73,33 +109,72 @@ def test_selinux_file_contexts_exclude(mocked_setfiles, tmp_path, stage_module):
         "file_contexts": "etc/selinux/thing",
         "exclude_paths": ["/sysroot"],
     }
-    stage_module.main(tmp_path, options)
+    args = {
+        "tree": f"{tmp_path}",
+        "options": options
+    }
+    stage_module.main(args)
 
     assert len(mocked_setfiles.call_args_list) == 1
     args, kwargs = mocked_setfiles.call_args_list[0]
-    assert args == (f"{tmp_path}/etc/selinux/thing", os.fspath(tmp_path), "")
+    assert args == (f"{tmp_path}/etc/selinux/thing", os.fspath(tmp_path), "/")
     assert kwargs == {"exclude_paths": [f"{tmp_path}/sysroot"]}
 
 
 @patch("osbuild.util.selinux.setfilecon")
 @patch("osbuild.util.selinux.setfiles")
 def test_selinux_labels(mocked_setfiles, mocked_setfilecon, tmp_path, stage_module):
     testutil.make_fake_input_tree(tmp_path, {
```

Comment:
(nitpick) for symetry with the test below we could do (but it is already a bit inconsistent in the tests so fine to keep as is):

```python
diff --git a/stages/test/test_selinux.py b/stages/test/test_selinux.py
index a5aadf9f..138a2123 100644
--- a/stages/test/test_selinux.py
+++ b/stages/test/test_selinux.py
@@ -124,6 +124,8 @@ def test_selinux_file_contexts_exclude(mocked_setfiles, tmp_path, stage_module):
 @patch("osbuild.util.selinux.setfilecon")
 @patch("osbuild.util.selinux.setfiles")
 def test_selinux_labels(mocked_setfiles, mocked_setfilecon, tmp_path, stage_module):
+    tree = tmp_path / "tree"
+
     testutil.make_fake_input_tree(tmp_path, {
         "/usr/bin/echo": "I'm only an imposter",
         "/sbin/sulogin": "I'm only an imposter",
@@ -137,7 +139,7 @@ def test_selinux_labels(mocked_setfiles, mocked_setfilecon, tmp_path, stage_modu
         }
     }
     args = {
-        "tree": f"{tmp_path}/tree",
+        "tree": tree,
         "options": options
     }
     stage_module.main(args)
@@ -145,8 +147,8 @@ def test_selinux_labels(mocked_setfiles, mocked_setfilecon, tmp_path, stage_modu
     assert len(mocked_setfiles.call_args_list) == 1
     assert len(mocked_setfilecon.call_args_list) == 2
     assert mocked_setfilecon.call_args_list == [
-        call(f"{tmp_path}/tree/usr/bin/echo", "system_u:object_r:bin_t:s0"),
-        call(f"{tmp_path}/tree/sbin/sulogin", "system_u:object_r:sulogin_exec_t:s0"),
+        call(f"{tree}/usr/bin/echo", "system_u:object_r:bin_t:s0"),
+        call(f"{tree}/sbin/sulogin", "system_u:object_r:sulogin_exec_t:s0"),
     ]
 
 

```